logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -2,8 +2,17 @@ package com.wire.kalium.logic.feature.call
 
 import com.wire.kalium.logic.data.message.Message
 import com.wire.kalium.logic.data.message.MessageContent
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.map
 
 expect class CallManager {
 
     suspend fun onCallingMessageReceived(message: Message, content: MessageContent.Calling)
+    val allCalls: StateFlow<List<Call>>
+}
+
+val CallManager.ongoingCalls get() = allCalls.map {
+    it.filter { call ->
+        call.status in listOf("Incoming", "Answered", "Established")
+    }
```

Comment:
This is currently not being used, but will be in the near future as we want to properly expose each types of calls, such as:
- Ongoing calls
- Missed / Closed calls
- and so on..